### ISSUE TITLE
no sound APTx (was working fine previously)

### ISSUE THREAD

**Issue by MeisterTea** [EHfive/pulseaudio-modules-bt#92]
Hi, my aptX playback doesn't work anymore. I can only use HSP/HFP

It was working fine previously.
Can you help me out at least to debug it ?

my versions: 

* Arch
* Linux 5.4.8.arch1-1
* pulseaudio-modules-bt-git-r143.b9d3bd3-1
* pulseaudio 13.0-2

Headset: Audio Technica ATH-M50xBT

avfinfo:

```
Connecting ... 
Stream End-Point #5: Audio Sink 
	Media Codec: Vendor Specific A2DP Codec
		Vendor ID 0x0000004f
		Vendor Specific Codec ID 0x0001
		Vendor Specific Data: 0x32
		Vendor Specific Value (aptX)
			Frequencies: 44.1kHz 48kHz 
			Channel modes: Stereo 
	Content Protection: SCMS-T
Stream End-Point #3: Audio Sink 
	Media Codec: MPEG24
		Object Types: MPEG-2 AAC LC MPEG-4 AAC LC 
		Frequencies: 8kHz 11.025kHz 12kHz 16kHz 22.05kHz 24kHz 32kHz 44.1kHz 48kHz 
		Channels: 1 2 
		Bitrate: 320000
		VBR: Yes
	Content Protection: SCMS-T
Stream End-Point #37: Audio Source 
	Media Codec: Vendor Specific A2DP Codec
		Vendor ID 0x0000004f
		Vendor Specific Codec ID 0x0001
		Vendor Specific Data: 0x32
		Vendor Specific Value (aptX)
			Frequencies: 44.1kHz 48kHz 
			Channel modes: Stereo 
	Content Protection: SCMS-T
Stream End-Point #35: Audio Source 
	Media Codec: MPEG24
		Object Types: MPEG-2 AAC LC MPEG-4 AAC LC 
		Frequencies: 8kHz 11.025kHz 12kHz 16kHz 22.05kHz 24kHz 32kHz 44.1kHz 48kHz 
		Channels: 1 2 
		Bitrate: 320000
		VBR: Yes
	Content Protection: SCMS-T
Stream End-Point #33: Audio Source 
	Media Codec: SBC
		Channel Modes: Mono DualChannel Stereo JointStereo
		Frequencies: 16Khz 32Khz 44.1Khz 48Khz 
		Subbands: 4 8
		Blocks: 4 8 12 16 
		Bitpool Range: 2-53
	Content Protection: SCMS-T
Stream End-Point #1: Audio Sink 
	Media Codec: SBC
		Channel Modes: Mono DualChannel Stereo JointStereo
		Frequencies: 16Khz 32Khz 44.1Khz 48Khz 
		Subbands: 4 8
		Blocks: 4 8 12 16 
		Bitpool Range: 2-53
	Content Protection: SCMS-T

```

**Comment by EHfive** [comment]
is your `ffmpeg` still installed

**Comment by EHfive** [comment]
If modules A2DP endpoints successfully registered
```
journalctl -fu bluetooth
-- Logs begin at Thu 2019-12-19 21:53:56 CST. --
1月 10 19:56:11 eh5-arch bluetoothd[884]: Endpoint registered: sender=:1.134 path=/MediaEndpoint/A2DPSink/VENDOR/APTXHD
1月 10 19:56:11 eh5-arch bluetoothd[884]: Endpoint registered: sender=:1.134 path=/MediaEndpoint/A2DPSink/VENDOR/APTX
1月 10 19:56:11 eh5-arch bluetoothd[884]: Endpoint registered: sender=:1.134 path=/MediaEndpoint/A2DPSink/AAC
1月 10 19:56:11 eh5-arch bluetoothd[884]: Endpoint registered: sender=:1.134 path=/MediaEndpoint/A2DPSink/SBC
```

Your pulseaudio bluetooth card & profiles info
```
$ LANGUAGE=C pactl list cards | grep bluez_card -A20
        Name: bluez_card.70_26_05_00_00_00
        Driver: module-bluez5-device.c
        Owner Module: 29
        Properties:
                device.description = "WH-1000XM2"
                device.string = "70:26:05:00:00:00"
                device.api = "bluez"
                device.class = "sound"
                device.bus = "bluetooth"
                device.form_factor = "headset"
                bluez.path = "/org/bluez/hci0/dev_70_26_05_00_00_00"
                bluez.class = "0x240404"
                bluez.alias = "WH-1000XM2"
                device.icon_name = "audio-headset-bluetooth"
                device.intended_roles = "phone"
        Profiles:
                headset_head_unit: Headset Head Unit (HSP/HFP) (sinks: 1, sources: 1, priority: 30, available: yes)
                a2dp_sink_sbc: High Fidelity Playback (A2DP Sink: SBC) (sinks: 1, sources: 0, priority: 40, available: yes)
                a2dp_sink_aac: High Fidelity Playback (A2DP Sink: AAC) (sinks: 1, sources: 0, priority: 40, available: yes)
                a2dp_sink_aptx: High Fidelity Playback (A2DP Sink: aptX) (sinks: 1, sources: 0, priority: 40, available: yes)
                a2dp_sink_aptx_hd: High Fidelity Playback (A2DP Sink: aptX HD) (sinks: 1, sources: 0, priority: 40, available: yes)
```

**Comment by MeisterTea** [comment]
ffmpeg is still installed: 1:4.2.1-4

the A2DP module endpoints are registered successfully
```
janv. 10 11:08:24 glados bluetoothd[1024]: Endpoint registered: sender=:1.911 path=/MediaEndpoint/A2DPSource/VENDOR/APTX
janv. 10 11:08:24 glados bluetoothd[1024]: Endpoint registered: sender=:1.911 path=/MediaEndpoint/A2DPSource/AAC
janv. 10 11:08:24 glados bluetoothd[1024]: Endpoint registered: sender=:1.911 path=/MediaEndpoint/A2DPSource/SBC
janv. 10 11:08:24 glados bluetoothd[1024]: Endpoint registered: sender=:1.911 path=/MediaEndpoint/A2DPSink/VENDOR/APTXHD
janv. 10 11:08:24 glados bluetoothd[1024]: Endpoint registered: sender=:1.911 path=/MediaEndpoint/A2DPSink/VENDOR/APTX
janv. 10 11:08:24 glados bluetoothd[1024]: Endpoint registered: sender=:1.911 path=/MediaEndpoint/A2DPSink/AAC
janv. 10 11:08:24 glados bluetoothd[1024]: Endpoint registered: sender=:1.911 path=/MediaEndpoint/A2DPSink/SBC
janv. 10 11:08:27 glados bluetoothd[1024]: /org/bluez/hci0/dev_00_0A_45_0B_1B_A5/sep5/fd10: fd(42) ready
janv. 10 12:50:03 glados bluetoothd[1024]: /org/bluez/hci0/dev_00_0A_45_0B_1B_A5/sep5/fd11: fd(42) ready
janv. 10 14:48:04 glados bluetoothd[1024]: /org/bluez/hci0/dev_00_0A_45_0B_1B_A5/sep5/fd12: fd(41) ready
```

It doesn't seem to be any any problem here as well
```
        Name: bluez_card.00_0A_45_0B_1B_A5
        Driver: module-bluez5-device.c
        Owner Module: 29
        Properties:
                device.description = "ATH-M50xBT    "
                device.string = "00:0A:45:0B:1B:A5"
                device.api = "bluez"
                device.class = "sound"
                device.bus = "bluetooth"
                device.form_factor = "headphone"
                bluez.path = "/org/bluez/hci0/dev_00_0A_45_0B_1B_A5"
                bluez.class = "0x240418"
                bluez.alias = "ATH-M50xBT    "
                device.icon_name = "audio-headphones-bluetooth"
        Profiles:
                headset_head_unit: Headset Head Unit (HSP/HFP) (sinks: 1, sources: 1, priority: 30, available: yes)
                a2dp_source_sbc: High Fidelity Capture (A2DP Source: SBC) (sinks: 0, sources: 1, priority: 20, available: yes)
                a2dp_source_aac: High Fidelity Capture (A2DP Source: AAC) (sinks: 0, sources: 1, priority: 20, available: yes)
                a2dp_source_aptx: High Fidelity Capture (A2DP Source: APTX) (sinks: 0, sources: 1, priority: 20, available: yes)
                a2dp_source_aptx_hd: High Fidelity Capture (A2DP Source: aptX HD) (sinks: 0, sources: 1, priority: 20, available: no)
                a2dp_sink_sbc: High Fidelity Playback (A2DP Sink: SBC) (sinks: 1, sources: 0, priority: 40, available: yes)

```

**Comment by EHfive** [comment]
might relates 
https://github.com/FFmpeg/FFmpeg/commit/675f62a202be7cfe9576b48679b8e6bc1183e84e & https://github.com/FFmpeg/FFmpeg/commit/98a257c3235bdc18151534134148845728418248

You could try downgrade `ffmpeg`. (using [downgrade](https://aur.archlinux.org/packages/downgrade/) , or fetch older package & install it)

**Comment by asavah** [comment]
I have build pulseaudio-modules-bt literally yesterday with ffmpeg git master - no issues here with aptx.

**Comment by MeisterTea** [comment]
Is there anything I can do to pinpoint where is the problem on this install?

**Comment by EHfive** [comment]
There might be useful info ouputed by `ffmpeg` in pulseaudio logs
```
$ systemctl --user stop pulseaudio.socket
$ systemctl --user stop pulseaudio.service
$ pulseaudio --verbose
```

**Comment by MeisterTea** [comment]
I got this:

```
I: [pulseaudio] main.c: setrlimit(RLIMIT_NICE, (31, 31)) failed: Operation not permitted
I: [pulseaudio] main.c: setrlimit(RLIMIT_RTPRIO, (9, 9)) failed: Operation not permitted
I: [pulseaudio] core-util.c: Successfully gained nice level -11.
I: [pulseaudio] main.c: This is PulseAudio 13.0-dirty
I: [pulseaudio] main.c: Page size is 4096 bytes
I: [pulseaudio] main.c: Machine ID is 3960d79e93ca47e486fb311eff91dbc5.
I: [pulseaudio] main.c: Session ID is 1.
I: [pulseaudio] main.c: Using runtime directory /run/user/1000/pulse.
I: [pulseaudio] main.c: Using state directory /home/chell/.config/pulse.
I: [pulseaudio] main.c: Using modules directory /usr/lib/pulse-13.0/modules.
I: [pulseaudio] main.c: Running in system mode: no
I: [pulseaudio] main.c: System supports high resolution timers
I: [pulseaudio] cpu-x86.c: CPU flags: CMOV MMX SSE SSE2 SSE3 SSSE3 SSE4_1 SSE4_2
I: [pulseaudio] svolume_mmx.c: Initialising MMX optimized volume functions.
I: [pulseaudio] remap_mmx.c: Initialising MMX optimized remappers.
I: [pulseaudio] svolume_sse.c: Initialising SSE2 optimized volume functions.
I: [pulseaudio] remap_sse.c: Initialising SSE2 optimized remappers.
I: [pulseaudio] sconv_sse.c: Initialising SSE2 optimized conversions.
I: [pulseaudio] svolume_orc.c: Initialising ORC optimized volume functions.
I: [pulseaudio] module-device-restore.c: Successfully opened database file '/home/chell/.config/pulse/3960d79e93ca47e486fb311eff91dbc5-device-volumes'.
I: [pulseaudio] module.c: Loaded "module-device-restore" (index: #0; argument: "").
I: [pulseaudio] module-stream-restore.c: Successfully opened database file '/home/chell/.config/pulse/3960d79e93ca47e486fb311eff91dbc5-stream-volumes'.
I: [pulseaudio] module.c: Loaded "module-stream-restore" (index: #1; argument: "").
I: [pulseaudio] module-card-restore.c: Successfully opened database file '/home/chell/.config/pulse/3960d79e93ca47e486fb311eff91dbc5-card-database'.
I: [pulseaudio] module.c: Loaded "module-card-restore" (index: #2; argument: "").
I: [pulseaudio] module.c: Loaded "module-augment-properties" (index: #3; argument: "").
I: [pulseaudio] module.c: Loaded "module-switch-on-port-available" (index: #4; argument: "").
I: [pulseaudio] (alsa-lib)utils.c: could not open configuration file /usr/share/alsa/ucm2/HDA Intel PCH/HDA Intel PCH.conf
I: [pulseaudio] (alsa-lib)parser.c: error: could not parse configuration for card HDA Intel PCH
I: [pulseaudio] (alsa-lib)main.c: error: failed to import HDA Intel PCH use case configuration -2
I: [pulseaudio] alsa-ucm.c: UCM not available for card HDA Intel PCH
I: [pulseaudio] (alsa-lib)control.c: Invalid CTL front:0
I: [pulseaudio] alsa-util.c: Unable to attach to mixer front:0: No such file or directory
I: [pulseaudio] alsa-util.c: Successfully attached to mixer 'hw:0'
I: [pulseaudio] alsa-util.c: Error opening PCM device iec958:0: No such file or directory
I: [pulseaudio] (alsa-lib)control.c: Invalid CTL front:0
I: [pulseaudio] alsa-util.c: Unable to attach to mixer front:0: No such file or directory
I: [pulseaudio] alsa-util.c: Successfully attached to mixer 'hw:0'
I: [pulseaudio] (alsa-lib)pcm_route.c: Found no matching channel map
I: [pulseaudio] alsa-util.c: Error opening PCM device surround21:0: Invalid argument
I: [pulseaudio] alsa-util.c: Failed to set hardware parameters on plug:surround40:0: Invalid argument
I: [pulseaudio] (alsa-lib)pcm_route.c: Found no matching channel map
I: [pulseaudio] alsa-util.c: Error opening PCM device surround41:0: Invalid argument
I: [pulseaudio] (alsa-lib)pcm_route.c: Found no matching channel map
I: [pulseaudio] alsa-util.c: Error opening PCM device surround50:0: Invalid argument
I: [pulseaudio] alsa-util.c: Failed to set hardware parameters on plug:surround51:0: Invalid argument
I: [pulseaudio] alsa-util.c: Failed to set hardware parameters on plug:surround71:0: Invalid argument
I: [pulseaudio] alsa-util.c: Error opening PCM device iec958:0: No such file or directory
I: [pulseaudio] (alsa-lib)pcm.c: Unknown PCM a52:0
I: [pulseaudio] alsa-util.c: Error opening PCM device a52:0: No such file or directory
I: [pulseaudio] (alsa-lib)pcm.c: Unknown PCM a52:0
I: [pulseaudio] alsa-util.c: Error opening PCM device a52:0: No such file or directory
I: [pulseaudio] (alsa-lib)pcm.c: Unknown PCM dca:0
I: [pulseaudio] alsa-util.c: Error opening PCM device dca:0: No such file or directory
I: [pulseaudio] (alsa-lib)control.c: Invalid CTL hdmi:0
I: [pulseaudio] alsa-util.c: Unable to attach to mixer hdmi:0: No such file or directory
I: [pulseaudio] alsa-util.c: Successfully attached to mixer 'hw:0'
I: [pulseaudio] (alsa-lib)control.c: Invalid CTL hdmi:0
I: [pulseaudio] alsa-util.c: Unable to attach to mixer hdmi:0: No such file or directory
I: [pulseaudio] alsa-util.c: Successfully attached to mixer 'hw:0'
I: [pulseaudio] (alsa-lib)control.c: Invalid CTL hdmi:0
I: [pulseaudio] alsa-util.c: Unable to attach to mixer hdmi:0: No such file or directory
I: [pulseaudio] alsa-util.c: Successfully attached to mixer 'hw:0'
I: [pulseaudio] (alsa-lib)pcm.c: Unknown PCM dcahdmi:0
I: [pulseaudio] alsa-util.c: Error opening PCM device dcahdmi:0: No such file or directory
I: [pulseaudio] (alsa-lib)control.c: Invalid CTL hdmi:0,1
I: [pulseaudio] alsa-util.c: Unable to attach to mixer hdmi:0,1: No such file or directory
I: [pulseaudio] alsa-util.c: Successfully attached to mixer 'hw:0'
I: [pulseaudio] (alsa-lib)conf.c: Unknown parameter 1
I: [pulseaudio] (alsa-lib)conf.c: Parse arguments error: No such file or directory
I: [pulseaudio] (alsa-lib)pcm.c: Unknown PCM plug:hdmi:0,1
I: [pulseaudio] alsa-util.c: Error opening PCM device plug:hdmi:0,1: No such file or directory
I: [pulseaudio] (alsa-lib)conf.c: Unknown parameter 1
I: [pulseaudio] (alsa-lib)conf.c: Parse arguments error: No such file or directory
I: [pulseaudio] (alsa-lib)pcm.c: Unknown PCM plug:hdmi:0,1
I: [pulseaudio] alsa-util.c: Error opening PCM device plug:hdmi:0,1: No such file or directory
I: [pulseaudio] (alsa-lib)pcm.c: Unknown PCM dcahdmi:0,1
I: [pulseaudio] alsa-util.c: Error opening PCM device dcahdmi:0,1: No such file or directory
I: [pulseaudio] (alsa-lib)control.c: Invalid CTL hdmi:0,2
I: [pulseaudio] alsa-util.c: Unable to attach to mixer hdmi:0,2: No such file or directory
I: [pulseaudio] alsa-util.c: Successfully attached to mixer 'hw:0'
I: [pulseaudio] (alsa-lib)control.c: Invalid CTL hdmi:0,2
I: [pulseaudio] alsa-util.c: Unable to attach to mixer hdmi:0,2: No such file or directory
I: [pulseaudio] alsa-util.c: Successfully attached to mixer 'hw:0'
I: [pulseaudio] (alsa-lib)control.c: Invalid CTL hdmi:0,2
I: [pulseaudio] alsa-util.c: Unable to attach to mixer hdmi:0,2: No such file or directory
I: [pulseaudio] alsa-util.c: Successfully attached to mixer 'hw:0'
I: [pulseaudio] (alsa-lib)pcm.c: Unknown PCM dcahdmi:0,2
I: [pulseaudio] alsa-util.c: Error opening PCM device dcahdmi:0,2: No such file or directory
I: [pulseaudio] (alsa-lib)control.c: Invalid CTL hdmi:0,3
I: [pulseaudio] alsa-util.c: Unable to attach to mixer hdmi:0,3: No such file or directory
I: [pulseaudio] alsa-util.c: Successfully attached to mixer 'hw:0'
I: [pulseaudio] (alsa-lib)control.c: Invalid CTL hdmi:0,3
I: [pulseaudio] alsa-util.c: Unable to attach to mixer hdmi:0,3: No such file or directory
I: [pulseaudio] alsa-util.c: Successfully attached to mixer 'hw:0'
I: [pulseaudio] (alsa-lib)control.c: Invalid CTL hdmi:0,3
I: [pulseaudio] alsa-util.c: Unable to attach to mixer hdmi:0,3: No such file or directory
I: [pulseaudio] alsa-util.c: Successfully attached to mixer 'hw:0'
I: [pulseaudio] (alsa-lib)pcm.c: Unknown PCM dcahdmi:0,3
I: [pulseaudio] alsa-util.c: Error opening PCM device dcahdmi:0,3: No such file or directory
I: [pulseaudio] (alsa-lib)control.c: Invalid CTL hdmi:0,4
I: [pulseaudio] alsa-util.c: Unable to attach to mixer hdmi:0,4: No such file or directory
I: [pulseaudio] alsa-util.c: Successfully attached to mixer 'hw:0'
I: [pulseaudio] (alsa-lib)control.c: Invalid CTL hdmi:0,4
I: [pulseaudio] alsa-util.c: Unable to attach to mixer hdmi:0,4: No such file or directory
I: [pulseaudio] alsa-util.c: Successfully attached to mixer 'hw:0'
I: [pulseaudio] (alsa-lib)control.c: Invalid CTL hdmi:0,4
I: [pulseaudio] alsa-util.c: Unable to attach to mixer hdmi:0,4: No such file or directory
I: [pulseaudio] alsa-util.c: Successfully attached to mixer 'hw:0'
I: [pulseaudio] (alsa-lib)pcm.c: Unknown PCM dcahdmi:0,4
I: [pulseaudio] alsa-util.c: Error opening PCM device dcahdmi:0,4: No such file or directory
I: [pulseaudio] alsa-util.c: Error opening PCM device hdmi:0,5: No such file or directory
I: [pulseaudio] alsa-util.c: Error opening PCM device hdmi:0,5: No such file or directory
I: [pulseaudio] alsa-util.c: Error opening PCM device hdmi:0,5: No such file or directory
I: [pulseaudio] (alsa-lib)pcm.c: Unknown PCM dcahdmi:0,5
I: [pulseaudio] alsa-util.c: Error opening PCM device dcahdmi:0,5: No such file or directory
I: [pulseaudio] alsa-util.c: Error opening PCM device hdmi:0,6: No such file or directory
I: [pulseaudio] alsa-util.c: Error opening PCM device hdmi:0,6: No such file or directory
I: [pulseaudio] alsa-util.c: Error opening PCM device hdmi:0,6: No such file or directory
I: [pulseaudio] (alsa-lib)pcm.c: Unknown PCM dcahdmi:0,6
I: [pulseaudio] alsa-util.c: Error opening PCM device dcahdmi:0,6: No such file or directory
I: [pulseaudio] alsa-util.c: Error opening PCM device hdmi:0,7: No such file or directory
I: [pulseaudio] alsa-util.c: Error opening PCM device hdmi:0,7: No such file or directory
I: [pulseaudio] alsa-util.c: Error opening PCM device hdmi:0,7: No such file or directory
I: [pulseaudio] (alsa-lib)pcm.c: Unknown PCM dcahdmi:0,7
I: [pulseaudio] alsa-util.c: Error opening PCM device dcahdmi:0,7: No such file or directory
I: [pulseaudio] module-card-restore.c: Restoring port latency offsets for card alsa_card.pci-0000_00_1f.3.
I: [pulseaudio] alsa-util.c: Successfully attached to mixer 'hw:0'
I: [pulseaudio] card.c: alsa_card.pci-0000_00_1f.3: active_profile: output:analog-stereo+input:analog-stereo
I: [pulseaudio] card.c: Created 0 "alsa_card.pci-0000_00_1f.3"
I: [pulseaudio] alsa-sink.c: Successfully opened device front:0.
I: [pulseaudio] alsa-sink.c: Selected mapping 'Analog Stereo' (analog-stereo).
I: [pulseaudio] alsa-sink.c: Successfully enabled mmap() mode.
I: [pulseaudio] (alsa-lib)control.c: Invalid CTL front:0
I: [pulseaudio] alsa-util.c: Unable to attach to mixer front:0: No such file or directory
I: [pulseaudio] alsa-util.c: Successfully attached to mixer 'hw:0'
I: [pulseaudio] module-device-restore.c: Restoring port for sink sink:alsa_output.pci-0000_00_1f.3.analog-stereo.
I: [pulseaudio] module-device-restore.c: Restoring volume for sink alsa_output.pci-0000_00_1f.3.analog-stereo: front-left: 32768 /  50%,   front-right: 32768 /  50%
I: [pulseaudio] module-device-restore.c: Restoring mute state for sink alsa_output.pci-0000_00_1f.3.analog-stereo: unmuted
I: [pulseaudio] sink.c: Created sink 0 "alsa_output.pci-0000_00_1f.3.analog-stereo" with sample spec s16le 2ch 48000Hz and channel map front-left,front-right
I: [pulseaudio] sink.c:     alsa.resolution_bits = "16"
I: [pulseaudio] sink.c:     device.api = "alsa"
I: [pulseaudio] sink.c:     device.class = "sound"
I: [pulseaudio] sink.c:     alsa.class = "generic"
I: [pulseaudio] sink.c:     alsa.subclass = "generic-mix"
I: [pulseaudio] sink.c:     alsa.name = "ALC294 Analog"
I: [pulseaudio] sink.c:     alsa.id = "ALC294 Analog"
I: [pulseaudio] sink.c:     alsa.subdevice = "0"
I: [pulseaudio] sink.c:     alsa.subdevice_name = "subdevice #0"
I: [pulseaudio] sink.c:     alsa.device = "0"
I: [pulseaudio] sink.c:     alsa.card = "0"
I: [pulseaudio] sink.c:     alsa.card_name = "HDA Intel PCH"
I: [pulseaudio] sink.c:     alsa.long_card_name = "HDA Intel PCH at 0xef228000 irq 129"
I: [pulseaudio] sink.c:     alsa.driver_name = "snd_hda_intel"
I: [pulseaudio] sink.c:     device.bus_path = "pci-0000:00:1f.3"
I: [pulseaudio] sink.c:     sysfs.path = "/devices/pci0000:00/0000:00:1f.3/sound/card0"
I: [pulseaudio] sink.c:     device.bus = "pci"
I: [pulseaudio] sink.c:     device.vendor.id = "8086"
I: [pulseaudio] sink.c:     device.vendor.name = "Intel Corporation"
I: [pulseaudio] sink.c:     device.product.id = "9d71"
I: [pulseaudio] sink.c:     device.product.name = "Sunrise Point-LP HD Audio"
I: [pulseaudio] sink.c:     device.form_factor = "internal"
I: [pulseaudio] sink.c:     device.string = "front:0"
I: [pulseaudio] sink.c:     device.buffering.buffer_size = "19200"
I: [pulseaudio] sink.c:     device.buffering.fragment_size = "4800"
I: [pulseaudio] sink.c:     device.access_mode = "mmap"
I: [pulseaudio] sink.c:     device.profile.name = "analog-stereo"
I: [pulseaudio] sink.c:     device.profile.description = "Analog Stereo"
I: [pulseaudio] sink.c:     device.description = "Built-in Audio Analog Stereo"
I: [pulseaudio] sink.c:     alsa.mixer_name = "Realtek ALC294"
I: [pulseaudio] sink.c:     alsa.components = "HDA:10ec0294,10431c20,00100004 HDA:8086280b,80860101,00100000"
I: [pulseaudio] sink.c:     module-udev-detect.discovered = "1"
I: [pulseaudio] sink.c:     device.icon_name = "audio-card-pci"
I: [pulseaudio] module-device-restore.c: Restoring mute state for source alsa_output.pci-0000_00_1f.3.analog-stereo.monitor: unmuted
I: [pulseaudio] source.c: Created source 0 "alsa_output.pci-0000_00_1f.3.analog-stereo.monitor" with sample spec s16le 2ch 48000Hz and channel map front-left,front-right
I: [pulseaudio] source.c:     device.description = "Monitor of Built-in Audio Analog Stereo"
I: [pulseaudio] source.c:     device.class = "monitor"
I: [pulseaudio] source.c:     alsa.card = "0"
I: [pulseaudio] source.c:     alsa.card_name = "HDA Intel PCH"
I: [pulseaudio] source.c:     alsa.long_card_name = "HDA Intel PCH at 0xef228000 irq 129"
I: [pulseaudio] source.c:     alsa.driver_name = "snd_hda_intel"
I: [pulseaudio] source.c:     device.bus_path = "pci-0000:00:1f.3"
I: [pulseaudio] source.c:     sysfs.path = "/devices/pci0000:00/0000:00:1f.3/sound/card0"
I: [pulseaudio] source.c:     device.bus = "pci"
I: [pulseaudio] source.c:     device.vendor.id = "8086"
I: [pulseaudio] source.c:     device.vendor.name = "Intel Corporation"
I: [pulseaudio] source.c:     device.product.id = "9d71"
I: [pulseaudio] source.c:     device.product.name = "Sunrise Point-LP HD Audio"
I: [pulseaudio] source.c:     device.form_factor = "internal"
I: [pulseaudio] source.c:     device.string = "0"
I: [pulseaudio] source.c:     module-udev-detect.discovered = "1"
I: [pulseaudio] source.c:     device.icon_name = "audio-card-pci"
I: [pulseaudio] alsa-sink.c: Using 4,0 fragments of size 4800 bytes (25,00ms), buffer size is 19200 bytes (100,00ms)
I: [pulseaudio] alsa-sink.c: Successfully enabled deferred volume.
I: [pulseaudio] alsa-sink.c: Hardware volume ranges from -181,50 dB to 0,00 dB.
I: [pulseaudio] alsa-sink.c: Fixing base volume to 0,00 dB
I: [pulseaudio] alsa-sink.c: Using hardware volume control. Hardware dB scale supported.
I: [pulseaudio] alsa-sink.c: Using hardware mute control.
I: [alsa-sink-ALC294 Analog] util.c: Successfully enabled SCHED_RR scheduling for thread, with priority 5.
I: [alsa-sink-ALC294 Analog] alsa-sink.c: Starting playback.
I: [pulseaudio] core.c: default_source: (unset) -> alsa_output.pci-0000_00_1f.3.analog-stereo.monitor
I: [pulseaudio] core.c: default_sink: (unset) -> alsa_output.pci-0000_00_1f.3.analog-stereo
I: [pulseaudio] alsa-source.c: Successfully opened device front:0.
I: [pulseaudio] alsa-source.c: Selected mapping 'Analog Stereo' (analog-stereo).
I: [pulseaudio] alsa-source.c: Successfully enabled mmap() mode.
I: [pulseaudio] (alsa-lib)control.c: Invalid CTL front:0
I: [pulseaudio] alsa-util.c: Unable to attach to mixer front:0: No such file or directory
I: [pulseaudio] alsa-util.c: Successfully attached to mixer 'hw:0'
I: [pulseaudio] module-device-restore.c: Restoring port for source source:alsa_input.pci-0000_00_1f.3.analog-stereo.
I: [pulseaudio] module-device-restore.c: Restoring volume for source alsa_input.pci-0000_00_1f.3.analog-stereo: front-left: 28013 /  43%,   front-right: 28013 /  43%
I: [pulseaudio] module-device-restore.c: Restoring mute state for source alsa_input.pci-0000_00_1f.3.analog-stereo: unmuted
I: [pulseaudio] source.c: Created source 1 "alsa_input.pci-0000_00_1f.3.analog-stereo" with sample spec s16le 2ch 48000Hz and channel map front-left,front-right
I: [pulseaudio] source.c:     alsa.resolution_bits = "16"
I: [pulseaudio] source.c:     device.api = "alsa"
I: [pulseaudio] source.c:     device.class = "sound"
I: [pulseaudio] source.c:     alsa.class = "generic"
I: [pulseaudio] source.c:     alsa.subclass = "generic-mix"
I: [pulseaudio] source.c:     alsa.name = "ALC294 Analog"
I: [pulseaudio] source.c:     alsa.id = "ALC294 Analog"
I: [pulseaudio] source.c:     alsa.subdevice = "0"
I: [pulseaudio] source.c:     alsa.subdevice_name = "subdevice #0"
I: [pulseaudio] source.c:     alsa.device = "0"
I: [pulseaudio] source.c:     alsa.card = "0"
I: [pulseaudio] source.c:     alsa.card_name = "HDA Intel PCH"
I: [pulseaudio] source.c:     alsa.long_card_name = "HDA Intel PCH at 0xef228000 irq 129"
I: [pulseaudio] source.c:     alsa.driver_name = "snd_hda_intel"
I: [pulseaudio] source.c:     device.bus_path = "pci-0000:00:1f.3"
I: [pulseaudio] source.c:     sysfs.path = "/devices/pci0000:00/0000:00:1f.3/sound/card0"
I: [pulseaudio] source.c:     device.bus = "pci"
I: [pulseaudio] source.c:     device.vendor.id = "8086"
I: [pulseaudio] source.c:     device.vendor.name = "Intel Corporation"
I: [pulseaudio] source.c:     device.product.id = "9d71"
I: [pulseaudio] source.c:     device.product.name = "Sunrise Point-LP HD Audio"
I: [pulseaudio] source.c:     device.form_factor = "internal"
I: [pulseaudio] source.c:     device.string = "front:0"
I: [pulseaudio] source.c:     device.buffering.buffer_size = "19200"
I: [pulseaudio] source.c:     device.buffering.fragment_size = "4800"
I: [pulseaudio] source.c:     device.access_mode = "mmap"
I: [pulseaudio] source.c:     device.profile.name = "analog-stereo"
I: [pulseaudio] source.c:     device.profile.description = "Analog Stereo"
I: [pulseaudio] source.c:     device.description = "Built-in Audio Analog Stereo"
I: [pulseaudio] source.c:     alsa.mixer_name = "Realtek ALC294"
I: [pulseaudio] source.c:     alsa.components = "HDA:10ec0294,10431c20,00100004 HDA:8086280b,80860101,00100000"
I: [pulseaudio] source.c:     module-udev-detect.discovered = "1"
I: [pulseaudio] source.c:     device.icon_name = "audio-card-pci"
I: [pulseaudio] alsa-source.c: Using 4,0 fragments of size 4800 bytes (25,00ms), buffer size is 19200 bytes (100,00ms)
I: [pulseaudio] alsa-source.c: Successfully enabled deferred volume.
I: [pulseaudio] alsa-source.c: Hardware volume ranges from -17,25 dB to 60,00 dB.
I: [pulseaudio] alsa-source.c: Fixing base volume to -60,00 dB
I: [pulseaudio] alsa-source.c: Using hardware volume control. Hardware dB scale supported.
I: [pulseaudio] alsa-source.c: Using hardware mute control.
I: [alsa-source-ALC294 Analog] util.c: Successfully enabled SCHED_RR scheduling for thread, with priority 5.
I: [alsa-source-ALC294 Analog] alsa-source.c: Starting capture.
I: [pulseaudio] core.c: default_source: alsa_output.pci-0000_00_1f.3.analog-stereo.monitor -> alsa_input.pci-0000_00_1f.3.analog-stereo
I: [pulseaudio] module.c: Loaded "module-alsa-card" (index: #6; argument: "device_id="0" name="pci-0000_00_1f.3" card_name="alsa_card.pci-0000_00_1f.3" namereg_fail=false tsched=no fixed_latency_range=no ignore_dB=no deferred_volume=yes use_ucm=yes avoid_resampling=no card_properties="module-udev-detect.discovered=1"").
I: [pulseaudio] module-udev-detect.c: Card /devices/pci0000:00/0000:00:1f.3/sound/card0 (alsa_card.pci-0000_00_1f.3) module loaded.
I: [pulseaudio] module-udev-detect.c: Found 1 cards.
I: [pulseaudio] module.c: Loaded "module-udev-detect" (index: #5; argument: "tsched=0").
I: [pulseaudio] module.c: Loaded "module-bluetooth-policy" (index: #7; argument: "").
I: [pulseaudio] module.c: Loaded "module-bluez5-discover" (index: #9; argument: "").
I: [pulseaudio] module.c: Loaded "module-bluetooth-discover" (index: #8; argument: "").
I: [pulseaudio] module.c: Loaded "module-dbus-protocol" (index: #10; argument: "").
I: [pulseaudio] module.c: Loaded "module-native-protocol-unix" (index: #11; argument: "").
I: [pulseaudio] module.c: Loaded "module-gsettings" (index: #12; argument: "").
I: [pulseaudio] module-default-device-restore.c: Restoring default sink 'bluez_sink.00_0A_45_0B_1B_A5.a2dp_sink'.
I: [pulseaudio] core.c: configured_default_sink: (unset) -> bluez_sink.00_0A_45_0B_1B_A5.a2dp_sink
I: [pulseaudio] module-default-device-restore.c: Restoring default source 'alsa_input.usb-Sennheiser_Communications_Sennheiser_USB_headset-00.mono-fallback'.
I: [pulseaudio] core.c: configured_default_source: (unset) -> alsa_input.usb-Sennheiser_Communications_Sennheiser_USB_headset-00.mono-fallback
I: [pulseaudio] module.c: Loaded "module-default-device-restore" (index: #13; argument: "").
I: [pulseaudio] module.c: Loaded "module-rescue-streams" (index: #14; argument: "").
I: [pulseaudio] module.c: Loaded "module-always-sink" (index: #15; argument: "").
I: [pulseaudio] module.c: Loaded "module-intended-roles" (index: #16; argument: "").
I: [pulseaudio] module.c: Loaded "module-suspend-on-idle" (index: #17; argument: "").
I: [pulseaudio] module.c: Loaded "module-console-kit" (index: #18; argument: "").
I: [pulseaudio] client.c: Created 0 "Login Session 1"
I: [pulseaudio] core.c: exit_idle_time: 20 -> 0
I: [pulseaudio] module.c: Loaded "module-systemd-login" (index: #19; argument: "").
I: [pulseaudio] module.c: Loaded "module-position-event-sounds" (index: #20; argument: "").
I: [pulseaudio] module.c: Loaded "module-role-cork" (index: #21; argument: "").
I: [pulseaudio] module.c: Loaded "module-filter-heuristics" (index: #22; argument: "").
I: [pulseaudio] module.c: Loaded "module-filter-apply" (index: #23; argument: "").
I: [pulseaudio] module.c: Loaded "module-switch-on-connect" (index: #24; argument: "").
I: [pulseaudio] main.c: Daemon startup complete.
I: [pulseaudio] module-device-restore.c: Storing volume/mute for device+port source:alsa_input.pci-0000_00_1f.3.analog-stereo:analog-input-internal-mic.
E: [pulseaudio] backend-ofono.c: Failed to register as a handsfree audio agent with ofono: org.freedesktop.DBus.Error.ServiceUnknown: The name org.ofono was not provided by any .service files
I: [pulseaudio] backend-native.c: doing listen
I: [pulseaudio] module-card-restore.c: Restoring port latency offsets for card bluez_card.00_0A_45_0B_1B_A5.
I: [pulseaudio] card.c: bluez_card.00_0A_45_0B_1B_A5: active_profile: a2dp_sink_aptx
I: [pulseaudio] card.c: Created 1 "bluez_card.00_0A_45_0B_1B_A5"
I: [pulseaudio] module-bluez5-device.c: Transport /org/bluez/hci0/dev_00_0A_45_0B_1B_A5/sep5/fd15 acquired: fd 38
I: [pulseaudio] module-device-restore.c: Restoring volume for sink bluez_sink.00_0A_45_0B_1B_A5.a2dp_sink: front-left: 34079 /  52%,   front-right: 34079 /  52%
I: [pulseaudio] module-device-restore.c: Restoring mute state for sink bluez_sink.00_0A_45_0B_1B_A5.a2dp_sink: muted
I: [pulseaudio] sink.c: Created sink 1 "bluez_sink.00_0A_45_0B_1B_A5.a2dp_sink" with sample spec s32le 2ch 48000Hz and channel map front-left,front-right
I: [pulseaudio] sink.c:     bluetooth.protocol = "a2dp_sink"
I: [pulseaudio] sink.c:     bluetooth.a2dp_codec = "aptX"
I: [pulseaudio] sink.c:     device.description = "ATH-M50xBT    "
I: [pulseaudio] sink.c:     device.string = "00:0A:45:0B:1B:A5"
I: [pulseaudio] sink.c:     device.api = "bluez"
I: [pulseaudio] sink.c:     device.class = "sound"
I: [pulseaudio] sink.c:     device.bus = "bluetooth"
I: [pulseaudio] sink.c:     device.form_factor = "headphone"
I: [pulseaudio] sink.c:     bluez.path = "/org/bluez/hci0/dev_00_0A_45_0B_1B_A5"
I: [pulseaudio] sink.c:     bluez.class = "0x240418"
I: [pulseaudio] sink.c:     bluez.alias = "ATH-M50xBT    "
I: [pulseaudio] sink.c:     device.icon_name = "audio-headphones-bluetooth"
I: [pulseaudio] module-device-restore.c: Restoring volume for source bluez_sink.00_0A_45_0B_1B_A5.a2dp_sink.monitor: front-left: 65536 / 100%,   front-right: 65536 / 100%
I: [pulseaudio] module-device-restore.c: Restoring mute state for source bluez_sink.00_0A_45_0B_1B_A5.a2dp_sink.monitor: unmuted
I: [pulseaudio] source.c: Created source 2 "bluez_sink.00_0A_45_0B_1B_A5.a2dp_sink.monitor" with sample spec s32le 2ch 48000Hz and channel map front-left,front-right
I: [pulseaudio] source.c:     device.description = "Monitor of ATH-M50xBT    "
I: [pulseaudio] source.c:     device.class = "monitor"
I: [pulseaudio] source.c:     device.string = "00:0A:45:0B:1B:A5"
I: [pulseaudio] source.c:     device.api = "bluez"
I: [pulseaudio] source.c:     device.bus = "bluetooth"
I: [pulseaudio] source.c:     device.form_factor = "headphone"
I: [pulseaudio] source.c:     bluez.path = "/org/bluez/hci0/dev_00_0A_45_0B_1B_A5"
I: [pulseaudio] source.c:     bluez.class = "0x240418"
I: [pulseaudio] source.c:     bluez.alias = "ATH-M50xBT    "
I: [pulseaudio] source.c:     device.icon_name = "audio-headphones-bluetooth"
I: [bluetooth] util.c: Successfully enabled SCHED_RR scheduling for thread, with priority 5.
I: [bluetooth] module-bluez5-device.c: Transport /org/bluez/hci0/dev_00_0A_45_0B_1B_A5/sep5/fd15 resuming
I: [bluetooth] module-bluez5-device.c: Changing bluetooth buffer size: Changed from 106496 to 14272
I: [pulseaudio] core.c: default_sink: alsa_output.pci-0000_00_1f.3.analog-stereo -> bluez_sink.00_0A_45_0B_1B_A5.a2dp_sink
I: [pulseaudio] module.c: Loaded "module-bluez5-device" (index: #25; argument: "path=/org/bluez/hci0/dev_00_0A_45_0B_1B_A5 autodetect_mtu=0 a2dp_config=""").
I: [pulseaudio] module-suspend-on-idle.c: Source alsa_input.pci-0000_00_1f.3.analog-stereo idle for too long, suspending ...
I: [alsa-source-ALC294 Analog] alsa-source.c: Device suspended...
I: [pulseaudio] module-suspend-on-idle.c: Sink alsa_output.pci-0000_00_1f.3.analog-stereo idle for too long, suspending ...
I: [alsa-sink-ALC294 Analog] alsa-sink.c: Device suspended...
I: [pulseaudio] module-suspend-on-idle.c: Sink bluez_sink.00_0A_45_0B_1B_A5.a2dp_sink idle for too long, suspending ...
I: [bluetooth] bluez5-util.c: Transport /org/bluez/hci0/dev_00_0A_45_0B_1B_A5/sep5/fd15 released
I: [pulseaudio] module-device-restore.c: Synced.
I: [pulseaudio] client.c: Created 1 "Native client (UNIX socket client)"
I: [pulseaudio] protocol-native.c: Got credentials: uid=1000 gid=1000 success=1
I: [pulseaudio] client.c: Created 2 "Native client (UNIX socket client)"
I: [pulseaudio] protocol-native.c: Got credentials: uid=1000 gid=1000 success=1
I: [pulseaudio] sink-input.c: Trying to change sample spec
I: [bluetooth] module-bluez5-device.c: Transport /org/bluez/hci0/dev_00_0A_45_0B_1B_A5/sep5/fd15 acquired: fd 43
I: [bluetooth] module-bluez5-device.c: Transport /org/bluez/hci0/dev_00_0A_45_0B_1B_A5/sep5/fd15 resuming
I: [pulseaudio] resampler.c: Forcing resampler 'copy', because of fixed, identical sample rates.
I: [pulseaudio] sink-input.c: Created input 0 "Playback" on bluez_sink.00_0A_45_0B_1B_A5.a2dp_sink with sample spec float32le 2ch 48000Hz and channel map front-left,front-right
I: [pulseaudio] sink-input.c:     application.icon_name = "google-chrome"
I: [pulseaudio] sink-input.c:     media.name = "Playback"
I: [pulseaudio] sink-input.c:     application.name = "Chromium"
I: [pulseaudio] sink-input.c:     native-protocol.peer = "UNIX socket client"
I: [pulseaudio] sink-input.c:     native-protocol.version = "33"
I: [pulseaudio] sink-input.c:     application.process.id = "252520"
I: [pulseaudio] sink-input.c:     application.process.user = "chell"
I: [pulseaudio] sink-input.c:     application.process.host = "glados"
I: [pulseaudio] sink-input.c:     application.process.binary = "chrome"
I: [pulseaudio] sink-input.c:     application.language = "en_US.UTF-8"
I: [pulseaudio] sink-input.c:     window.x11.display = ":0"
I: [pulseaudio] sink-input.c:     application.process.machine_id = "3960d79e93ca47e486fb311eff91dbc5"
I: [pulseaudio] sink-input.c:     application.process.session_id = "1"
I: [pulseaudio] sink-input.c:     module-stream-restore.id = "sink-input-by-application-name:Chromium"

```

Is there anything out of the ordinary ? :/
